setup.py

```diff
@@ -13,7 +13,7 @@
     # "xevan_hash" package is required to sync Xuez network.
     # "groestlcoin_hash" package is required to sync Groestlcoin network.
     install_requires=['aiorpcX>=0.7.1,<0.8', 'attrs',
-                      'plyvel', 'pylru', 'aiohttp >= 2'],
+                      'plyvel', 'pylru', 'aiohttp >= 2', 'blake256'],
```

Comment:
Sorry, no coin-specific packages can go here.  Please remove this and just add it to .travis.yml instead so it works for the tests.  But anyway - this seems to have nothing to do with decred testnet.

Reply:
I just removed the 'blake256' from the setup.py, and also some styling errors on coin.py. Regarding your comment about nothing to do with decred testnet, the functionality still the same but the parameters are different for the testnet (XPUB_VERBYTES,XPRV_VERBYTES, etc...) is that what you are referencing to or was that blake256 on the setup.py?
Please let me know if you want me to do anything else.